x-pack/plugins/grokdebugger/server/lib/license_pre_routing_factory/__tests__/license_pre_routing_factory.js

```diff
@@ -34,32 +34,28 @@ describe('license_pre_routing_factory', () => {
         };
       });
 
-      it ('replies with 403', (done) => {
+      it ('replies with 403', async () => {
         const licensePreRouting = licensePreRoutingFactory(mockServer);
         const stubRequest = {};
-        licensePreRouting(stubRequest, (response) => {
-          expect(response).to.be.an(Error);
-          expect(response.isBoom).to.be(true);
-          expect(response.output.statusCode).to.be(403);
-          done();
-        });
+        const response = await licensePreRouting(stubRequest);
+        expect(response).to.be.an(Error);
+        expect(response.isBoom).to.be(true);
+        expect(response.output.statusCode).to.be(403);
       });
     });
 
-    describe('isAvailable is true', () => {
+    describe('isAvailable is true', async () => {
       beforeEach(() => {
         mockLicenseCheckResults = {
           isAvailable: true
         };
       });
 
-      it ('replies with nothing', (done) => {
+      it ('replies with nothing', async () => {
         const licensePreRouting = licensePreRoutingFactory(mockServer);
         const stubRequest = {};
-        licensePreRouting(stubRequest, (response) => {
-          expect(response).to.eql(Boom.forbidden());
-          done();
-        });
+        const response = licensePreRouting(stubRequest);
```

Comment:
This needs to be awaited.

Reply:
The factory is now a regular function, not async. It only used callbacks before b/c of the reply function, not because what it looksup is asynchronous.